docs/content/integrations/embedded-elt.mdx

```diff
@@ -0,0 +1,126 @@
+| --------------------------------------------------------------------------- | ------------------------------------------------------------------------------ |
+| <PyObject module="dagster_embedded_elt.sling" object="build_sling_asset" /> | The core Sling asset factory for building syncs                                |
+| <PyObject module="dagster_embedded_elt.sling" object="SlingResource" />     | The Sling Resource used for handing credentials to databases and object stores |
+
+---
+
+## Getting Started
+
+To get started with `dagster-embedded-elt` and Sling, first familiarize yourself with <a href="https://docs.slingdata.io/sling-cli/running-tasks">Sling's documentation</a>
+
+First, setup a <PyObject module="dagster-embedded-elt.sling" object="SlingResource" /> which defines the source and destination credentials. The <PyObject module="dagster-embedded-elt.sling" object="SlingResource" /> requires both a <PyObject module="dagster-embedded-elt.sling" object="SlingSourceConnection" /> and a <PyObject module="dagster-embedded-elt.sling" object="SlingTargetConnection" />.
+
+You can provide either a connection string or a dictionary of connection parameters to each of these classes.
+
+For details on what connection parameters Sling accepts for each integration, see the <a href="https://docs.slingdata.io/connections/database-connections">Sling Connections</a> page.
+
+To create an Asset that syncs between two connections, you can use the <PyObject module="dagster-embedded-elt.sling" object="build_sling_asset" /> factory.
+
+---
+
+## Setting up a Sling Resource
+
+A Sling Resource is a Dagster Resource that contains references to both a Source Connection and a Target Connection. Sling is versatile in what a source or destination can represent. You can provide arbitrary keywords to the <PyObject module="dagster-embedded-elt.sling" object="SlingSourceConnection" /> and <PyObject module="dagster-embedded-elt.sling" object="SlingTargetConnection" /> classes.
+
+The types and parameters for each connection are defined by <a href="https://docs.slingdata.io/connections/database-connections">Sling's Connections</a>.
+
+The simplest connection is a file connection, which can be defined as:
+
+```python
+from dagster_embedded_elt.sling import SlingSourceConnection
+source = SlingSourceConnection(type="file")
+```
+
+Note that no path is required, as that is provided by the asset itself.
+
+```python
+asset_def = build_sling_asset(
+    asset_spec=AssetSpec("my_file"),
+    source_stream=f"file://{path_to_file}",
+    ...
+)
+
+```
+
+For database connections, you can provide a connection string or a dictionary of keyword arguments. For example, to connect to a SQLite database, you can provide an path to the database using the `instance` keyword, which is specified on the <a href="https://docs.slingdata.io/connections/database-connections/sqlite">Sqlite Connection</a> page.
```

Comment:
```suggestion
For database connections, you can provide a connection string or a dictionary of keyword arguments. For example, to connect to a SQLite database, you can provide a path to the database using the `instance` keyword, which is specified in [Sling's SQLite connection](https://docs.slingdata.io/connections/database-connections/sqlite) documentation.
```

Fixes a typo. I also updated this to use Markdown syntax for consistency, and to specifically state that the link goes to Sling's documentation. Less of a surprise for those who use screen readers.